src/unix/fs.c

```diff
@@ -125,6 +125,21 @@
   }                                                                           \
   while (0)
 
+#define POST0                                                                 \
```

Comment:
`req->result` is a `union { int errno; uv_os_fd_t fd; }` for the `uv_fs_open` case. On Unix, this worked out because `int fd >= 0`. On Windows, this doesn't really work, since `fd` is a pointer. I could perhaps just replace all `POST` calls with `POST0`, since there is at least a couple of other functions where the result value is `union { int errno, ssize_t nbytes }` and the `POST` macro here is silently truncating the return value on Unix as well.

I'm certainly open to other suggestions on how to refactor to handle this better.